src/mappings/entities/mapTransferManager.ts

```diff
@@ -19,25 +28,58 @@ const handleTransferManagerAdded = async (blockId: string, params: Codec[]) => {
 
   const ticker = serializeTicker(rawTicker);
   const { type, value } = getTransferManagerValue(rawManager);
+  const id = `${ticker}/${type}/${value}`;
+  const complianceType =
+    type === TransferRestrictionTypeEnum.Percentage
+      ? TransferComplianceTypeEnum.MaxInvestorOwnership
+      : TransferComplianceTypeEnum.MaxInvestorCount;
+
+  const promises = [
+    TransferManager.create({
+      id,
+      assetId: ticker,
+      type,
+      value,
+      exemptedEntities: [],
+      createdBlockId: blockId,
+      updatedBlockId: blockId,
+    }).save(),
+    TransferCompliance.create({
+      id,
```

Comment:
TransferCompliance id in these cases should be mapped as ticker/opType. Also, StatType creation promise should be pushed before creating TransferCompliance

Reply:
I moved it up since thats the intention, but if there is a race condition I am not sure dispatching the promise earlier is enough to guarantee it gets reflected in the DB first.

Hopefully it isn't an issue.